xapian-bindings/android/CROSS_COMPILE

```diff
@@ -75,3 +111,9 @@ After pushing all the files to the device, you can run test as follows:
 adb shell
 cd <dst_on_device>
 sh runtest ./apitest
+
+2)Now to test your bindings, create a empty project in your android studio and copy the files in the examples folder to your respective project folder. Add the xapianlib.aar in your project's libs folder and add the following line in your app's build.gradle file:
+
+compile(name:'xapianlib',ext:'aar')
```

Comment:
Can we put the Gradle file, and anything else required to build this, within `xapian-bindings/android` so people don't have to make a new project anywhere for the purpose of testing things? I don't know what other files are required to make it work, but ideally we'd drive that all from `make` instead of having to use Android Studio.